DolbyIO/Source/Private/DolbyIOObserver.cpp

```diff
@@ -14,9 +14,9 @@ UDolbyIOObserver::UDolbyIOObserver()
 
 void UDolbyIOObserver::InitializeComponent()
 {
-	if (UGameInstance* GameInstance = GetWorld()->GetGameInstance())
+	if (GetWorld() && GetWorld()->GetGameInstance())
```

Comment:
We may store it in some UWorld* variable if we're going to GetWorld() so many times.